src/Onebip/DateTime/UTCDateTimeRange.php

```diff
@@ -168,6 +168,11 @@ public function direction()
         }
     }
 
+    public function sizeInDays()
```

Comment:
what do you think about a nane like `distanceInDays()`?

Reply:
We are talking about `range`, `size` sound better to me. Maybe `interval` is better with dates.

Reply:
or simply `differenceBetweenDatesInDays`

Reply:
It's a public function i would like to keep the name nice and short. I think `intervalInDays()` it the best candidate right now.

Reply:
Good @lucacervello  you are right